backend/controllers/search.js

```diff
@@ -37,22 +40,22 @@ const externalSearchFactory = (
   }
 
   const results = async (req, res) => {
-    const { user, groupCode, roleCode, size, page } = req.query
+    const { user, groupCode, roleCode, size, page, offset } = req.query
 
     const pageSize = (size && parseInt(size, 10)) || 20
     const pageNumber = (page && parseInt(page, 10)) || 0
+    const pageOffset = (offset && parseInt(offset, 10)) || 0
 
     try {
-      const searchResults = await searchApi(res.locals, user, groupCode, roleCode, pageNumber, pageSize)
-      const pageResponse = contextProperties.getPageable(res.locals)
+      const searchResults = await searchApi(res.locals, user, groupCode, roleCode, pageNumber, pageSize, pageOffset)
 
-      res.render('externalSearchResults.njk', {
+      res.render(dpsSearch ? 'dpsSearchResults.njk' : 'externalSearchResults.njk', {
         searchTitle,
         searchUrl,
         maintainUrl,
         results: searchResults,
         pagination: paginationService.getPagination(
-          pageResponse,
+          pagingApi(res.locals),
```

Comment:
need different methods to get at the paging information, since one contains `page` and the other `offset`